tests/actions.go

```diff
@@ -1106,8 +1035,26 @@ func (oa *operatorActions) tidbMembersReadyFn(tc *v1alpha1.TidbCluster) (bool, e
 			ns, tidbSetName, tidbSet.Status.ReadyReplicas, tidbSet.Status.Replicas)
 		return false, nil
 	}
-	if c, ok := getComponentContainer(tidbSet); !ok || tc.Spec.TiDB.Image != c.Image {
-		glog.Infof("statefulset: %s/%s .spec.template.spec.containers[name=tikv].image(%s) != %s",
+
+	if upgradePaused() {
+
+		time.Sleep(5 * time.Minute)
```

Comment:
The upgrade process take more than 5 minutes, so you must sleep more than it, for exmaple 10 minutes.

Reply:
upgradePause make sure that the last TiDB pod is upgraded and healthy, So I think 5 minute is enough.